Dockerfile

```diff
@@ -23,10 +23,12 @@ RUN mkdir /projects ${HOME} && \
 ENV GLIBC_VERSION=2.31-r0 \
     ODO_VERSION=v1.2.1 \
     OC_VERSION=4.3.3 \
-    KUBECTL_VERSION=v1.18.3
+    KUBECTL_VERSION=v1.18.3 \
+    KUBECTX_VERSION=0.8.0
 
-# plugin executes the commands relying on Bash
-RUN apk add --update --no-cache bash && \
+RUN apk add --update --no-cache bash bash-completion ncurses pkgconfig && \
+    ln -sf /bin/bash /bin/sh && \
```

Comment:
Is this necessary if we set `/bin/bash` as the login shell in `/etc/passwd`?

Reply:
Not really necessary if we talk about terminal establishing with che-machine-exec.
Without it - user with Pod-Contianer-Terminal get sh with that changes - bash... <- But I should check to confirm that it's true )